### ISSUE TITLE
is there anyway to include fields in Screen Option?

### ISSUE THREAD

**Issue by gurumark** [rilwis/meta-box#623]
Is there any way to include custom metabox fields in edit Screen Option tab? For example, you have many fields and you want to give users to remove the fields that they do not need. 
Currently, it happens at the metabox level, not the field level.

**Comment by rilwis** [comment]
Do you mean add option to Screen Options to toggle fields? If so, there's no such feature for now, but that sounds interesting.

**Comment by gurumark** [comment]
Yes. I think it will be useful if users could turn on and off fields using Screen Options vs disabling whole block. I looked around and could not find anything in WP Codex. It could be implemented using JS.

**Comment by funkedgeek** [comment]
Couldn't you just separate your fields in different meta boxes and hide the meta boxes they don't need?  I can easily see the screen options area getting cluttered with dozens of fields.  Meta boxes can already be hidden so I don't think this feature is needed

**Comment by gurumark** [comment]
The metaboxes are meant to be displaying interrelated data. If you hide the metabox, it will hide everything inside. We want users to have ability to disable what they want at the field level. The cluttering issue is not really a big deal. In addition, it will be useful to be able to select which ones go in there.
Thanks for your response in any case.

**Comment by funkedgeek** [comment]
Not sure it's the job of Screen Options, since that is more for showing and hiding sections/modules instead of individual items

**Comment by gurumark** [comment]
There are many plugins implementing enabling/disabling fields/columns in Screen options. Besides I don't think Slug or Author can be considered a section.
